core-bundle/src/Resources/contao/library/Contao/Model.php

```diff
@@ -1072,7 +1072,7 @@ protected static function find(array $arrOptions)
 		}
 
 		$objStatement = static::preFind($objStatement);
-		$objResult = $objStatement->execute($arrOptions['value']);
+		$objResult = $objStatement->execute($arrOptions['value'] ?? null);
 
 		if ($objResult->numRows < 1)
```

Comment:
if `$objResult` is null, access to an undefined property will raise a warning in PHP 8.

```suggestion
		if (null !== $objResult && $objResult->numRows < 1)
```

Reply:
why would `$objStatement->execute()` return `null`? It's a database query, not a model collection 😉

Reply:
Ops, I've missed the parenthesis 🙈 😂